subsystems/win/basesrv/dosdev.c

```diff
@@ -492,9 +492,9 @@ CSR_API(BaseSrvDefineDosDevice)
     BOOLEAN HandleTarget;
     BOOLEAN Broadcast = FALSE;
     BOOLEAN IsGlobal = FALSE;
-    ULONG CchLengthLeft;
-    ULONG CchLength;
-    ULONG TargetLength;
+    SIZE_T CchLengthLeft;
+    SIZE_T CchLength;
+    SIZE_T TargetLength;
```

Comment:
Later on in the code, these variables are used to initialize `Length` and `MaximumLength` members of `UNICODE_STRING` variables (which are `USHORT`), so there may require additional treatment to avoid new warnings.